package day40_exception;

public class UsingThrows {

    public static void main(String[] args) {

        try{
            Thread.sleep(2000);
        } catch (Exception e) {

        }

    }

}
